subsys/net/lib/ftp_client/Kconfig

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig FTP_CLIENT
+	bool "FTP client"
+
+if FTP_CLIENT
+
+config FTP_SERVER_PORT
+	int "FTP service port on remote host"
+	default 21
+
+config FTP_USER_ANONYMOUS
```

Comment:
Actually, now I see these configs are not even used by the library, they should not be present here. Probably would fit in the SLM configuration.

Reply:
It's defined for lib's user to use when needed. The non-anonymous username/password should be provided by the lib user, or it could choose to use this definition. Define as config item in case anonymous access does not use "anonymous" user name.